source/unified-test-format/tests/invalid/outcomeCollectionData-additionalProperties.yml

```diff
@@ -0,0 +1,24 @@
+description: "outcomeCollectionData-additionalProperties"
+
+schemaVersion: "1.5"
```

Comment:
This can also be "1.0", as it would have been invalid based on the original schema when `collectionData` was a shared definition.